packages/orbit-components/package.json

```diff
@@ -11,7 +11,8 @@
     "cy:dev": "vite cypress/integration --logLevel silent",
     "cy:run": "start-server-and-test cy:dev http-get://localhost:3000 'cypress run'",
     "cy:open": "start-server-and-test cy:dev http-get://localhost:3000 'cypress open'",
-    "build": "ts-node --esm config/build/index.mts",
+    "build": "yarn tailwind && ts-node --esm config/build/index.mts",
```

Comment:
you do need to modify that command here, as you already added that as a step to build inside index.mts :)

Reply:
Yes, it was outdated as I did not add that change 👍 well spotted though